src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -384,55 +364,47 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     )
   }, [percentageSelection, position])
 
-  const poolAssetFeeAssetDustAmountCryptoPrecision = useMemo(() => {
-    if (!poolAssetFeeAsset) return '0'
-    const dustAmountCryptoBaseUnit =
-      THORCHAIN_SAVERS_DUST_THRESHOLDS_CRYPTO_BASE_UNIT[poolAssetFeeAsset?.assetId] ?? '0'
-    return fromBaseUnit(dustAmountCryptoBaseUnit, poolAssetFeeAsset?.precision)
-  }, [poolAssetFeeAsset])
-
-  const runeDustAmountCryptoPrecision = useMemo(() => {
-    if (!runeAsset) return '0'
-    const dustAmountCryptoBaseUnit =
-      THORCHAIN_SAVERS_DUST_THRESHOLDS_CRYPTO_BASE_UNIT[runeAsset?.assetId] ?? '0'
-    return fromBaseUnit(dustAmountCryptoBaseUnit, runeAsset?.precision)
-  }, [runeAsset])
-
-  // We reuse lending utils here since all this does is estimating fees for a given withdrawal amount with a memo
-  // It's not going to be 100% accurate for EVM chains as it doesn't calculate the cost of depositWithExpiry, but rather a simple send,
-  // however that's fine for now until accurate fees estimation is implemented
+  const memo = useMemo(() => {
+    const withdrawalBps = bnOrZero(percentageSelection).times(100).toFixed()
```

Comment:
we could use a `convertPercentageToBasisPoints`, similar to the existing `convertBasisPointsToPercentage`

Reply:
🧠  71adbe8be